css/deploy.css

```diff
@@ -29,6 +29,10 @@
  * ---------------------------------------------------------------------
  */
 
+.select2-container--default .select2-results__group {
+   font-weight: 600;
```

Comment:
Won't that affect core and other plugins select2 dropdowns?

Reply:
Not a big fan about it too, better would be adding a !important flag on GLPI core, since the Deploy action includes ``ext-js`` which overrides this default GLPI CSS and not apply it. Actually this is the same value of GLPI core, I'll open a PR on GLPI Core for it.